tslearn/bases.py

```diff
@@ -233,7 +233,12 @@ def from_json(cls, path):
         for k in model['model_params'].keys():
```

Comment:
Where in the code is from_json exactly used? Is it called by some dependency such as keras? tf.keras has a `model_from_json` as well, perhaps that could be used to solve this "hack"?

Reply:
It's used when one wants to load a model from disk.
`model_from_json` loads the model structure but not the weights. I think we need this hack.